docs/pages/components/date-range-picker/en-US/index.md

```diff
@@ -198,6 +198,7 @@ Learn more in [Accessibility](/guide/accessibility).
 | size                 | 'lg' &#124; 'md' &#124; 'sm' &#124; 'xs' `('md')`                                | A picker can have different sizes                                                    |
 | toggleAs             | ElementType `('a')`                                                              | You can use a custom element for this component                                      |
 | value                | [ValueType](#code-ts-value-type-code)                                            | Value (Controlled)                                                                   |
+| renderTitle          | (date: Date) => React.ReactNode                                                  | Custom render for month's title                                                      |
```

Comment:
The table is sorted by prop name, `renderTitle` should come before `renderValue`.

I recommend you to use the plugin for vscode.
https://github.com/simonguo/vscode-markdown-table-sort

Reply:
Fixed